### PR TITLE
Update l10n/pl

### PR DESCRIPTION
**Pull request by splewako**
This update syncs translations with desktop and adds (hopefully all) missing strings.

### CODE REVIEW

File: l10n/pl/add-on.properties [hunk 1]
```diff
@@ -2,26 +2,139 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-initiate_call_button_label2=Gotowi do rozmowy?
+clientSuperShortname=Firefox Hello
```
Comment (Standard8) [inline l10n/pl/add-on.properties:5]: This should just be 'Hello'

Comment (splewako) [inline l10n/pl/add-on.properties:5]: Why? For some wordplay that totally doesn't work in Polish "Firefox Hello" should work much better.

Comment (Standard8) [inline l10n/pl/add-on.properties:5]: Ok, it wasn't clear that was the case and it hadn't just been missed.

**Comment by splewako** [comment]
Thank you!

**Pull request merged**